commons-geometry-spherical/src/test/java/org/apache/commons/geometry/spherical/twod/RegionBSPTree2STest.java

```diff
@@ -626,6 +626,17 @@ public void testGeographicMap() {
         checkBarycenterConsistency(france);
     }
 
+    @Test
+    public void testCircleToPolygonBarycenter() {
+        Point2S center = Point2S.of(1.0, 1.0);
+        RegionBSPTree2S ccw = circleToPolygon(center, 0.0001, 200, false);
+        SphericalTestUtils.assertPointsEq(center, ccw.getBarycenter(), BARYCENTER_EPS);
+
+        // clockwise should just be antipodal for the center
+        RegionBSPTree2S cw = circleToPolygon(center, 0.0001, 200, true);
+        SphericalTestUtils.assertPointsEq(center.antipodal(), cw.getBarycenter(), BARYCENTER_EPS);
```

Comment:
This test is currently failing!

Reply:
After the fix in https://github.com/apache/commons-geometry/pull/53,

`Expected point to be equivalent to (4.141592653589792, 2.141592653589793) but was (5.8884091858825425, 1.8372466013631619)`